modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -192,7 +198,106 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_worker-machi
 ----
 +
 Because you create and manage the worker machines yourself, you do not need to initialize these machines.
+//The current AWS UPI method removes all of the `99_openshift-cluster-api_worker-machineset-* files. What is different about the Local Zones that means that we should retain them? Should we remove whatever the installation program creates and then specifically create this new file?
 endif::aws,azure,ash,gcp[]
+ifdef::localzone[]
+. Create the machine set manifests for the worker nodes in your Local Zone.
+.. Export a variable to define the instance type for the worker machine to deploy on the Local Zone subnet by running the following command:
++
+[source,terminal]
+----
+$ export INSTANCE_TYPE="<instance_type>" <1>
+----
+<1> Set `<instance_type>` to a tested instance type, such as `c5d.2xlarge`.
+
+.. Store the AMI ID as a local variable by running the following command:
++
+[source,terminal]
+----
+$ export AMI_ID=$(grep ami \
+  openshift/99_openshift-cluster-api_worker-machineset-0.yaml \
+  | tail -n1 | awk '{print$2}')
+----
+
+.. Store the AMI ID as a local variable by running the following command:
++
+[source,terminal]
+----
+$ export SUBNET_ID=$(aws cloudformation describe-stacks \
+  --stack-name "<subnet_stack_name>" \ <1>
+  | jq -r .Stacks[0].Outputs[0].OutputValue)
+----
+<1> For `<subnet_stack_name>`, specify the name of the subnet stack that you created.
+
+.. Create the worker manifest file for the Local Zone that your VPC uses by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF > openshift/99_openshift-cluster-api_worker-machineset-nyc1.yaml
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: ${CLUSTER_ID}
```

Comment:
The [step](https://github.com/mtulio/mtulio.labs/blob/article-ocp-aws-lz/docs/articles/ocp-aws-local-zones-day-0.md#create-the-installer-manifests-) to populate the `${CLUSTER_ID}` wasn't moved. I think it can be the first step of step "Create the machine set manifests for the worker nodes in your Local Zone."

```
export CLUSTER_ID="$(awk '/infrastructureName: / {print $2}' manifests/cluster-infrastructure-02-config.yml)"
```